disposebag/src/main/kotlin/com/kizitonwose/android/disposebag/DisposeBagPlugins.kt

```diff
@@ -0,0 +1,7 @@
+package com.kizitonwose.android.disposebag
+
+import android.arch.lifecycle.Lifecycle
+
+object DisposeBagPlugins {
```

Comment:
I don't really see the need for a new file just for the Plugins since there isn't much in it right now. I think keeping everything that has to do with `DisposeBag` class in the `DisposeBag` file is better, for visibility.

Reply:
Like i commented earlier it should not be about visibility but separation of concerns. Despite the fact that this file is small, we should not wait till we need to extend it before we separate files. Doing this from the onset shows intent, we easily add to this file in the future without refactoring.